django/__init__.py

```diff
@@ -22,3 +36,7 @@ def setup(set_prefix=True):
             '/' if settings.FORCE_SCRIPT_NAME is None else settings.FORCE_SCRIPT_NAME
         )
     apps.populate(settings.INSTALLED_APPS)
+
+    # Only now that setup was successful, we prevent later recalls
```

Comment:
Not sure this comment really helps since we already know what `is_ready` is for

Reply:
Ok changed